compatibility-matrix/compatibility-matrix.md

```diff
@@ -2,18 +2,18 @@
 
 ## Version compatibility matrix
 
-| Vendor \ Linkurious version|    2.10.x     | 3.0.0 - 3.0.6 | 3.0.7 - 3.0.x |     3.1.x     |      4.0.x      |
-|:---------------------------|:-------------:|:-------------:|:-------------:|:-------------:|:---------------:|
-| [Neo4j][a]                 | 3.2 - 4.2[^1] | 3.2 - 4.2[^1] |   3.2 - 4.4   |  4.0.2 - 4.4  | 4.1.0 - 5.x[^2] |
-| [Neo4 Aura][b]             |      Yes      |      Yes      |      Yes      |      Yes      |      Yes        |
-| [Elasticsearch][c]         | 1.4 - 6.8[^3] | 6.0 - 7.x[^3] | 6.0 - 7.x[^3] | 6.0 - 7.x     |      7.x        |
-| [JanusGraph][d]            | 0.1.x - 0.2.x |      No       |      No       |      No       |      No         |
-| [Azure Cosmos DB][e]       |      Yes      |      Yes      |      Yes      |      Yes      |      Yes        |
-| [Memgraph][f]              |      No       |      No       |      No       |      No       | 2.4 - 2.7[^4]   |
-| [Amazon Neptune][g]        |      No       |      No       |      No       |      No       |   1.2.0[^4]     |
-| [MySQL][h]                 |   5.6 - 8.0   |   5.6 - 8.0   |   5.6 - 8.0   |   5.6 - 8.0   |   5.6 - 8.0     |
-| [MariaDB][i]               |  10.1 - 10.5  |  10.1 - 10.5  |  10.1 - 10.5  |  10.1 - 10.5  |  10.1 - 10.5    |
-| [Microsoft SQL Server][j]  |  2014 - 2019  |  2014 - 2019  |  2014 - 2019  |  2014 - 2019  |  2014 - 2019    |
+| Vendor \ Linkurious version|    2.10.x     | 3.0.0 - 3.0.6 | 3.0.7 - 3.0.x |     3.1.x     |      4.0.x      |      4.0.7      |
+|:---------------------------|:-------------:|:-------------:|:-------------:|:-------------:|:---------------:|:---------------:|
+| [Neo4j][a]                 | 3.2 - 4.2[^1] | 3.2 - 4.2[^1] |   3.2 - 4.4   |  4.0.2 - 4.4  | 4.1.0 - 5.x[^2] | 4.1.0 - 5.x[^2] |
+| [Neo4 Aura][b]             |      Yes      |      Yes      |      Yes      |      Yes      |      Yes        |       Yes       |
+| [Elasticsearch][c]         | 1.4 - 6.8[^3] | 6.0 - 7.x[^3] | 6.0 - 7.x[^3] | 6.0 - 7.x     |      7.x        |   7.x - 8.0.6   |
```

Comment:
Here, I also think we should reuse the same model than the one `6.0 - 7.x` one. 

Not specifying the highest supported minor version is OK as we don't expect any breaking change in a minor version. On the other hand, using something like `7.0 - 8.6.0` would force us to regularly update this matrix.

```suggestion
| [Elasticsearch][c]         | 1.4 - 6.8[^3] | 6.0 - 7.x[^3] | 6.0 - 7.x[^3] | 6.0 - 7.x     |      7.x        |    7.0 - 8.x    |
```